packages/cf-component-page/test/__snapshots__/Page.js.snap

```diff
@@ -2,7 +2,7 @@
 
 exports[`should render 1`] = `
 <article
-  className="cf-page"
+  className=""
```

Comment:
this is an indication that something is broken, fela should always assign a className value.